### DISCUSSION TITLE
Update Sandboxed Program / Executeable ?

### DISCUSSION THREAD

**Discussion by horseshelpinghorses** [sandboxie-plus/Sandboxie#2566] [Q&A Feedback] [answered]
I have several copies of the same program, but with different data, running sandboxed.  There is an update to the program; how can I install it separately in each sandbox?

**Comment by offhub** [comment, marked as answer]
[Backup your data!] You can right-click on the installer and select 'Run Sandboxed > Select the sandbox' to install to your preferred sandbox.

OR

Extract the contents of the installer (or install it to different sandbox) and do a copy-paste files/folders to your preferred sandbox.
